### ISSUE TITLE
When the command is long enough you can't see the end of it, making it impossible to extend the command further

### ISSUE THREAD

**Issue by johnib** [warpdotdev/Warp#2460]
### Describe the bug

I often use the shell capabilities to take a base64 string, decode and decompress it.
For example:

https://app.warp.dev/block/5TIfWvew9W231AApvExgqP

```bash
echo 'H4sIAJXUumMAA4uuVkovzUxRslJKSUq0MDO0SNK1NE5L0TVJMTbUTbIwNtNNTbFISTU1NDUyMLZUqo0FAAuuqG8xAAAA' | base64 -d | gunzip
[{"guid":"dba8618b-93fd-4d31-b836-ed8de5152039"}]%                                                                                                                                      
```

However, when the base64 string is long enough, the window only shows me the beginning of it, not the end.. so I can't write the latter part of the command: `| base64 -d | gunzip` (for instance):

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/5327862/211202290-632a31bc-ad39-4ce1-a670-51ca6091572a.png">

Scrolling down takes me 1-2 cm down, but then it auto-scrolls back to the top of the command.
So basically I can't paste long inputs and pipe them through different commands.

### To Reproduce

as described above.. 

### Expected behaviour

Show the end of the command, while hiding the beginning of it.

### Screenshots

see main description 

### Operating System

MacOS

### OS Version

13.1

### Shell Version

zsh 5.8.1 (x86_64-apple-darwin22.0)

### Warp Version

v0.2023.01.03.08.03.stable_01

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Warp Internal (ignore): linear-label:b8107fdf-ba31-488d-b103-d271c89cac3e

None

**Comment by dannyneira** [comment]
Thanks for this, I know we're working on some improvements to the max scroll/block size and this may be related.
We'll post any updates on here.

**Comment by sandzhaj** [comment]
confirm
